### ISSUE TITLE
Quick start doc

### ISSUE THREAD

**Issue by drewnoakes** [microsoft/MSBuildSdks#108]
It'd be useful to show how to use these SDKs. 

I wanted to use `Microsoft.Build.NoTargets` so just put that string into the `Sdk` attribute of my `Project`, which then gave:

> The project file cannot be opened by the project system, because it is missing some critical imports or the referenced SDK cannot be found.
>
> Detailed Information:
> C:\Program Files\dotnet\sdk\3.0.100-preview6-012264\Sdks\Microsoft.Build.NoTargets\Sdk not found. Check that a recent enough .NET Core SDK is installed and/or increase the version specified in global.json.

After some experimentation it seems that this works:

```xml
<Project>
  <Sdk Name="Microsoft.Build.NoTargets" Version="1.0.73" />
  ...
```

However I'm not really sure that's correct. I saw mention of `Directory.Build.props` in some discussion on the issue tracker, but again it's not clear what to put in that file.

A few additional paragraphs on the root README would be helpful.

**Comment by guillaume86** [comment]
You can use Id/Version for the Project Sdk attribute:

    <Project Sdk="Microsoft.Build.NoTargets/1.0.73">

**Comment by jeffkl** [comment]
The main README has a link to how SDKs are resolved: https://docs.microsoft.com/en-us/visualstudio/msbuild/how-to-use-project-sdk?view=vs-2019#how-project-sdks-are-resolved

I didn't want to replicate too much of the documentation here.  Please feel free to contribute to the docs so that they make more sense to people just getting started.

**Comment by drewnoakes** [comment]
Thanks. I didn't see that one-word link as it was buried in a section about what they _are_. Those docs also don't demonstrate the name/version syntax described by @guillame86 above.

Completely understand the desire to limit duplicating docs. #112 shows the change I would have liked to see on the README which gives a quick guide on how to use them, and I think makes the link to the documentation easier to find. I'll also make a PR to the above docs to show the name/version syntax and tag you there for your review.

**Comment by jeffkl** [comment]
Closed by #112
